pkg/console/starter/starter.go

```diff
@@ -118,6 +128,11 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		oauthinformers.WithTweakListOptions(tweakListOptionsForOAuth),
 	)
 
+	consoleInformers := consoleinformers.NewSharedInformerFactory(
```

Comment:
I'm considering calling this `consoleExtensionInformers`, just for clarity.  Its hard to keep track of all the different configs.

Reply:
`consoleCLIDownloadsInformers`... per our conversation

Reply:
👍